src/python/grpcio/grpc/aio/__init__.py

```diff
@@ -55,7 +55,7 @@
 from ._metadata import Metadata
 from ._server import server
 from ._typing import ChannelArgumentType
-
+EOF = EOF
```

Comment:
What's the old `EOF` we're using in `__all__`?

Reply:
```
from grpc._cython.cygrpc import EOF
```